frontend/src/scenes/dashboard/DashboardHeader.tsx

```diff
@@ -76,12 +80,13 @@ export function DashboardHeader(): JSX.Element | null {
         exportOptions.push({
             export_format: ExporterFormat.JSON,
             export_context: {
-                localData: asDashboardTemplate,
+                localData: JSON.stringify(asDashboardTemplate, null, 4),
```

Comment:
```suggestion
                localData: JSON.stringify(asDashboardTemplate),
```
nit: no spaces necessary, as we're sending this to the api (?)